compiler/src/dotty/tools/dotc/transform/CacheAliasImplicits.scala

```diff
@@ -46,19 +46,36 @@ class CacheAliasImplicits extends MiniPhase with IdentityDenotTransformer { this
 
```

Comment:
The documentation comment of this class still states that aliases start as defs, it should be updated to describe how things work now.